src/Uno.UWP/UI/ViewManagement/UISettings.cs

```diff
@@ -1,27 +1,74 @@
 #pragma warning disable 108 // new keyword hiding
 #pragma warning disable 114 // new keyword hiding
+using System.Collections.Concurrent;
 using Uno;
+using Uno.Helpers.Theming;
+using Windows.Foundation;
+using Windows.UI.Core;
 
 namespace Windows.UI.ViewManagement
 {
-	[NotImplemented]
-	public  partial class UISettings 
-	{
-		[NotImplemented]
-		public bool AnimationsEnabled
+	/// <summary>
+	/// Contains a set of common app user interface settings and operations.
+	/// </summary>
+	/// <remarks>Events on this class are fired as long as the instance is alive.
```

Comment:
It's the same for UWP?

Reply:
Yes, the same. If you don't keep strong reference to the UISettings, it will not raise events and gets collected :-D .